From 5430eb2bff62e829d79ccd311b846dd820f3a5d9 Mon Sep 17 00:00:00 2001
From: Xiaoyin Liu <xiaoyinl@users.noreply.github.com>
Date: Wed, 28 Oct 2020 05:32:52 -0400
Subject: [PATCH] remove unused JAR_PATH variable

---
 Ghidra/Features/GhidraServer/build.gradle | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Ghidra/Features/GhidraServer/build.gradle b/Ghidra/Features/GhidraServer/build.gradle
index f695e257a..4ddc10613 100644
--- a/Ghidra/Features/GhidraServer/build.gradle
+++ b/Ghidra/Features/GhidraServer/build.gradle
@@ -80,7 +80,6 @@ task generateGhidraServerClasspath {
 			int idx = 0
 			configurations.runGhidraServer.each { jarFile ->	
 				if (!jarFile.path.contains("/libsForBuild/")) {			
-					def JAR_PATH = rootProject.relativePath(jarFile.absolutePath)
 					File distPath = file("/")
 					String resPath
 					for (Configuration conf : project.getConfigurations()) {
-- 
2.28.0

